php/libraries/Project.class.inc

```diff
@@ -194,23 +194,23 @@ class Project
      * @note If the database value is null then the return value will be the sum of
      * all this project's subproject recruitment target.
      *
-     * @return string This project's name
+     * @return int This project's name
```

Comment:
Probably the scope of another PR but should not that be "@return in This project's **target**

Reply:
Yes. I mostly ignored the comments/descriptions I noticed that made my head hurt in this PR for the sake of my sanity.

But I just removed this description entirely (other than the type annotation), since it doesn't really add any value that isn't already in the comment/function name.